lib/command/Command.js

```diff
@@ -207,7 +215,19 @@ class Command {
         return !req;
     }
 
-    cooldownCheck(userID) {
+    cooldownExclusionCheck(msg) {
+        if(this.cooldownExclusions.userIDs.indexOf(msg.author.id) > -1) return true;
+        if(this.cooldownExclusions.guildIDs.indexOf(msg.channel.guild.id) > -1) return true;
+        if(this.cooldownExclusions.channelIDs.indexOf(msg.channel.id) > -1) return true;
```

Comment:
Perhaps reorder these three checks? I'd imagine channel exclusions are going to be more likely than user exclusions, which will be more likely than guild exclusions.